scaleout/studio/templates/celery-worker-deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+    kompose.cmd: kompose convert
+    kompose.version: 1.20.0 ()
+  creationTimestamp: null
+  labels:
+    io.kompose.service: {{ .Release.Name }}-celery-worker
+  name: {{ .Release.Name }}-celery-worker
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      annotations:
+        kompose.cmd: kompose convert
+        kompose.version: 1.20.0 ()
+      creationTimestamp: null
+      labels:
+        io.kompose.service: {{ .Release.Name }}-celery-worker
+    spec:
+      containers:
+      - args:
+        - sh
+        - ./scripts/run_worker.sh
+        env:
+        - name: GET_HOSTS_FROM
+          value: dns
+        image: {{ .Values.celery.image.repository }}
+        imagePullPolicy: Always
+        name: {{ .Release.Name }}-celery-worker
+        resources: {}
+        volumeMounts:
+        - mountPath: /app/celery
+          name: data-repositories
+      {{- if .Values.imagePullSecret }}
+      imagePullSecrets:
+        - name: {{ .Values.imagePullSecret }}
+      {{- end }}
+      restartPolicy: Always
+      volumes:
+      - name: data-repositories
```

Comment:
This claim is reserved for the git-repos. Add another pvc claim template and use that new name. Is the mountpoint in need of a storage space? Otherwise you can skip this entirely.